web/src/main/java/com/epam/esm/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.esm.exception;
+
+import org.springframework.context.MessageSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.Collections;
+import java.util.Locale;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
```

Comment:
You could use `@ResponseBody` on class lvl + `@ResponseStatus` on methods and get rid of `ResponseEntity` in every method inside